js/PageLevelProgressIndicatorView.js

```diff
@@ -76,10 +76,14 @@ class PageLevelProgressIndicatorView extends Backbone.View {
   }
 
   checkCompletion() {
-    const percentage = this.setPercentageComplete();
```

Comment:
Minor: I think the `return percentage` can be dropped from the setPercentageComplete function now as it isn't used anywhere